com.unity.formats.fbx/Documentation~/export-record-in-play-mode-recorder-window.md

```diff
@@ -0,0 +1,25 @@
+# Record FBX animations via the Recorder window
+
+Use the Recorder window to record a GameObject's animation to FBX within an arbitrary time or frame interval in Play mode.
+
+>**Note:** This scenario requires to have the [Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest) package installed in addition to the FBX Exporter package.
+
+To record an FBX animation via the Recorder window:
+
+1. Open the Recorder window: select **Window** > **General** > **Recorder** > **Recorder Window**.
```

Comment:
could we show the image here too of the recorder window (i.e. the image from the recorder properties), like how `Export GameObjects to FBX` section shows the options